### ISSUE TITLE
stlib may fail to login when reusing login session to login on another account

### ISSUE THREAD

**Issue by ShyPixie** [ShyPixie/stlib#10]
<s>These passwords are failing to login:
```
5ZN2"MSP1$tXcBB7|&83G+'+~VU|r&1jar!gT8;8FSKAB;Tk3WekCDVG!IqAP@W?
9_B$9x>%t}m:phwH,b!uD1l,d@u4Vl![YeL}gmgQ}.jT=guL;s{HO8P:.-#u.\V@
78iUv8/NO^FUu\0Rz\p!XRVe1qbc9'8`zL+"`khUnrYX/!@$|EUf%j9aklx:5e=Q
```

Maybe something related to rsa encryption or base64 encoding
That will need debugging</s>

**Comment by ShyPixie** [comment]
I found out that it's not related to encryption or encoding, there's something wrong when reusing login session to login on another account, even if cookies and data is reset steam server returns false

**Comment by ShyPixie** [comment]
fixed by b56eda2ac8c65b9ce7a2e3ad58001e0f3bf186cf
